content/articles/create-a-registration-form-using-react.js-and-connecting-it-with-firebase/index.md

```diff
@@ -319,118 +369,126 @@ function RegistrationForm() {
+ 
 `npm install firebase`
 
-- To add the package into your react app we have to install firebase package over there. Now using terminal or command prompt go to the directory where your project is and run this command. 
- 
 ![package](./package.png)
 
-Create a new file with the name "firebase.js" in the Src folder where we will copy the code. 
+Create a new file with the name "firebase.js" in the Src folder where you will copy the code. 
 
-we have initialize the fire base. Now we will import data base in our code. 
+You have to initialize the firebase. Now you will import the database into your code. 
 - For importing the data write the following statement
 
 `import { getDatabase } from "firebase/database"`
 
-- In order to use this data base in our registration form component we will use  export statement. Add the following statements in the code.
+- To use this database in your registration form component you will use an export statement. Add the following statements in the code.
 `export const database = getDatabase(app);`
 
-In the registration form component import the following files
+In the registration, form component import the following files
 
 `import {database} from '../firebase'
 import {ref,push,child,update} from "firebase/database";`
 
-- To store the input entered by the user we need to add the following code into the handle Submit function we made earlier 
+- To store the input entered by the user you need to add the following code in the handle Submit function you made earlier 
 
-```java script
-let obj = {
+```javascript
+const handleSubmit = () =>{
+    let obj = {
             firstName : firstName,
             lastName:lastName,
             email:email,
             password:password,
             confirmPassword:confirmPassword,
         }       
-        const newPostKey = push(child(ref(database), 'posts')).key;
-        const updates = {};
-        updates['/' + newPostKey] = obj
-        return update(ref(database), updates);
+    const newPostKey = push(child(ref(database), 'posts')).key;
+    const updates = {};
+    updates['/' + newPostKey] = obj
+    return update(ref(database), updates);
+}
 ```
 
-Npw we have completed our steps successfully. 
+Now you have completed your steps successfully. 
 
-****Now we will see how it work****
+****Now you will see how it work****
 
-We will fill the details in the form then click on register .
+You will fill in the details in the form then click on register.
 
 ![Details](./Details.png)
 
-When we go to your firebase base account and check your Realtime database the your input is available there as shown in below pic.
+For checking the input you can see that from your firebase account. It will appear as shown below.
 
 ![Datainfirebase](./Datainfirebase.png)
 
-Congratulation you have successfully learn how to make registration form and connect it to firebase to store real time data.
+***Conclusion***
```

Comment:
This should be under H3 tag.
Please properly rearrange or rephrase this section so that it make sense to your readers.

Reply:
h3 is the main heading all the subheadings are made to h4 
Done